packages/client/components/DiscussionThreadListEmptyState.tsx

```diff
@@ -132,14 +132,14 @@ const DiscussionThreadListEmptyState = (props: Props) => {
       }
     }
   })
-  if (!meeting) return null
```

Comment:
The issue is that this component is not used only in retro meetings, so if meeting is not retro we don't show beautiful empty state at all